### DISCUSSION TITLE
Unable to obtain the number of successful writes

### DISCUSSION THREAD

**Discussion by majormj** [sqlalchemy/sqlalchemy#9629] [Usage Questions]
### Describe the bug

# Unable to obtain the number of successful writes
# <sqlalchemy.engine.cursor.CursorResult object at 0x00000282AB2D8D00>

### SQLAlchemy Version in Use

5.2

### DBAPI (i.e. the database driver)

pymysql

### Database Vendor and Major Version

MySQL 5.2

### Python Version

python 3.11

### Operating system

Windows

### To Reproduce

```python
from sqlalchemy import create_engine, Table, Column, Integer, String, MetaData
from sqlalchemy import  text

engine = create_engine('mysql+pymysql://root:**@mysql.xxxxxxx.cn:22564/data')

sql = f"replace INTO table_name (col1, col2, col3) VALUES ('value1', 'value2', 'value3')"
with engine.begin() as conn:
s = conn.execute(text(sql))
print(s)
# Unable to obtain the number of successful writes
# <sqlalchemy.engine.cursor.CursorResult object at 0x00000282AB2D8D00>
```

### Error

```
# <sqlalchemy.engine.cursor.CursorResult object at 0x00000282AB2D8D00>
```

**Comment by CaselIT** [comment]
Hi,

You may be looking for `result.rownumber`
